GoogleTestRunner/GoogleTestDiscoverer.fs

```diff
@@ -9,6 +9,7 @@ module Constants =
     [<Literal>]
     let identifierUri = "executor://GoogleTestRunner/v1";
     let gtestListTests = "--gtest_list_tests"
+    let gtestHelp = "-h"
```

Comment:
Also, what will happen with versions prior to 1.3.0?

https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#running-test-programs-advanced-options